### PR TITLE
added animations

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

**Comment by kcbreuil** [comment]
@asasmith been trying to debug this. Looks like the animations (two of the 4) render a div container to sit in them. I am not quite sure how to target this plain div. Do you have any insight on this? I have a feeling it's how Leslie exported them.

File: src/components/heroAnimation.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            setActiveAnimationFile(mobileHero);
+        } else if (window.innerWidth >= 769 || window.innerWidth < 1280) {
+            setActiveAnimationFile(heroOne);
+        } else {
+            setActiveAnimationFile(heroAnimation);
+        }
+    };
+
+    useEffect(() => {
+        window.addEventListener('resize', handleResize);
```
Comment (asasmith) [inline src/components/heroAnimation.js:26]: You should be able to move this into the `useEffect` below. Make sure to remove the event listener in the return function

File: src/components/initiativesAnimation.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        },
+        accelerate: {
+            id: `acc`,
+            heading: `Accelerate`,
+            body: `Building and executing campaigns to guide our initiatives through the necessary chanels of implementation, be they statuatory, regulatory, or at the agency level. `,
+        },
+    };
+    return (
+        <InitiativesWrapper className="initiative-container">
+            <span className="icons">
```
Comment (asasmith) [inline src/components/initiativesAnimation.js:101]: these should probably just be `div`'s

File: src/components/indexHero.js [hunk 3]
```diff
@@ -22,27 +26,33 @@ const Body = styled.p`
...
-    ];
+export default function IndexHero({ cta }) {
+    // const sources = [
+    //     mobileHero.childImageSharp.fluid,
+    //     {
+    //         ...heroImage.childImageSharp.fluid,
+    //         media: `(min-width: 768px)`,
+    //     },
+    // ];
+
```
Comment (asasmith) [inline src/components/indexHero.js:37]: remove commented code

File: src/components/fullWidthImageWithText.js [hunk 4]
```diff
@@ -22,21 +22,20 @@ const StyledBody = styled.p`
...
-    image,
     hasCurvedTransition,
     hasTopTransition,
 }) {
-    const {
-        childImageSharp: { fluid },
-    } = image;
+    // const {
+    //     childImageSharp: { fluid },
+    // } = image;
```
Comment (asasmith) [inline src/components/fullWidthImageWithText.js:30]: remove commented code

File: src/components/fullWidthImageWithText.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import tw from 'twin.macro';
 
-import Img from 'gatsby-image';
+// import Img from 'gatsby-image';
```
Comment (asasmith) [inline src/components/fullWidthImageWithText.js:6]: remove commented code

File: src/components/fullWidthImageWithText.js [hunk 6]
```diff
@@ -22,21 +22,20 @@ const StyledBody = styled.p`
...
+    // } = image;
 
     return (
         <StyledSection>
             {hasTopTransition && <ParallelTransition />}
             {hasCurvedTransition && <CurvedTransition />}
             <StyledHeading>{heading}</StyledHeading>
             <StyledBody>{body}</StyledBody>
-            <Img fluid={fluid} />
+            {/* <Img fluid={fluid} /> */}
```
Comment (asasmith) [inline src/components/fullWidthImageWithText.js:38]: remove commented code

File: src/components/fullWidthImageWithText.js [hunk 7]
```diff
@@ -49,7 +48,7 @@ FullWidthImageWithText.defaultProps = {
 FullWidthImageWithText.propTypes = {
     body: PropTypes.string.isRequired,
     heading: PropTypes.string.isRequired,
-    image: PropTypes.object.isRequired,
+    // image: PropTypes.object.isRequired,
```
Comment (asasmith) [inline src/components/fullWidthImageWithText.js:51]: remove commented code

File: src/components/initiativesAnimation.js [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+
+import accelerate from '../animations/Accelerate.json';
+import cultivate from '../animations/Cultivate.json';
+import icons from '../animations/Icons.json';
+import incubate from '../animations/Incubate.json';
+
+import InitiativesText from './initiativesText';
+
+const InitiativesWrapper = styled.div`
+    ${tw`grid grid-cols-5 grid-rows-5 justify-center`}
```
Comment (asasmith) [inline src/components/initiativesAnimation.js:14]: 5 rows doesn't work on larger screens, there's too much space between the wrapper element and the next section. If you need that many rows for smaller screens use whichever use one of the media query prefixes

File: src/components/initiativesAnimation.js [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+    }
+    #acc {
+        grid-column-start: 3;
+        grid-column-end: 5;
+    }
+    .accelerate {
+        grid-column-start: 2;
+        grid-row-start: 4;
+        grid-row-end: 5;
+    }
```
Comment (asasmith) [inline src/components/initiativesAnimation.js:49]: I'm not a fan of this naming convention. I wouldn't truncate names like `incu`. I'd also add something that makes it very clear if that styling is focused on an animation or text element `-animation` or `-text`.

File: src/components/initiativesAnimation.js [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+            id: `incu`,
+            heading: `Incubate`,
+            body: `Leveraging the policy expertise of our Strategic Council, technical team, and campaign partners to transform novel policy ideas into actionable policy initiatives. `,
+        },
+        accelerate: {
+            id: `acc`,
+            heading: `Accelerate`,
+            body: `Building and executing campaigns to guide our initiatives through the necessary chanels of implementation, be they statuatory, regulatory, or at the agency level. `,
+        },
+    };
```
Comment (asasmith) [inline src/components/initiativesAnimation.js:93]: same comment about class/id naming convention

File: src/components/initiativesText.js [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+
+const StyledHeading = styled.h2`
+    ${tw`font-wt-bold text-ts-h4 text-brand-1`}
+`;
+
+const StyledBody = styled.p`
+    ${tw`text-lg`}
+`;
+
+export default function InitiativesText({ id, heading, body }) {
```
Comment (asasmith) [inline src/components/initiativesText.js:18]: alphabetize these props please

**Review by asasmith — CHANGES_REQUESTED** [review]

**Review by asasmith — APPROVED** [review]

**Pull request merged**